docs/yum.8

```diff
@@ -858,12 +858,12 @@ the installroot (and thus. will be empty, before creation).
 .br
 Configuration Option: \fBinstallroot\fP
 .IP "\fB\-\-enablerepo=repoidglob\fP"
-Enables specific repositories by id or glob that have been disabled in the 
+Temporarily enables specific repositories by id or glob that have been disabled in the 
```

Comment:
Based on my comment in the related BZ https://bugzilla.redhat.com/show_bug.cgi?id=1573154#c5, I propose that we omit this hunk and just use the very first one (`[command]` -> `COMMAND`).

Reply:
Yeah, I just noticed they are separate commits. Thanks, I'll do the honors then.